view/frontend/web/js/view/payment/method-renderer/multishipping/adyen-klarna-method.js

```diff
@@ -26,17 +26,14 @@ define([
 
     return Component.extend({
         defaults: {
-            template: 'Adyen_Payment/payment/multishipping/klarna-form'
-        },
-        initialize: function () {
-            // Retrieve adyen payment methods
-            adyenPaymentService.retrievePaymentMethods();
-            this._super();
```

Comment:
If you don't call ` this._super();`, frontend component will not `initialize` the parent frontend component. This results in failing to load Adyen payment component.